### DISCUSSION TITLE
Markdown source code mode

### DISCUSSION THREAD

**Discussion by soulchild** [outline/outline#5751] [Ideas]
By far the number one feature request expressed by our team for our internal Outline wiki is the ability to edit documents in Markdown source code mode. I am often struggling with the WYSIWYG editor as well, finding myself deleting whole blocks of content and recreating it because I'm unable to get the formatting right.

Is that something you have thought about implementing?

**Comment by tommoor** [comment]
Given the way collaborative editing works it's not possible to have some editors viewing one version of a document and other editors viewing another – collaborative editing would have to be disabled which would be a shame. Markdown is also too restrictive in terms of layout and attributes that are available, we've been gradually moving away for it in order to support the many modern editor features that are requested such as rich embeds, resizing of objects, multiple colors etc.

If you have trouble using the editor please do file bugs with videos and we can fix those individual issues. Of course you can also write markdown in an external editor and paste it in.
